exporter/logzioexporter/from_domain.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2018 Uber Technologies, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package logzioexporter
+
+import (
+	"strings"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
+// NewFromDomain creates FromDomain used to convert model span to db span
+func NewFromDomain(allTagsAsObject bool, tagKeysAsFields []string, tagDotReplacement string) FromDomain {
```

Comment:
what about the tests?

Reply:
> what about the tests?

@yurishkuro I copy unit test here. should we add more?🤔